app/src/main/java/com/xa/xpensauditor/MonthWiseVis.java

```diff
@@ -140,30 +143,30 @@ public void onDataChange(@NonNull DataSnapshot datasnapshot) {
                             dumm = snapshot.child("Day").getValue().toString();
                             dumm1 = Integer.valueOf(dumm);
                             System.out.println("dummmmm   " + dumm1 + "    " + dayyy);
-                            if (dayyy == dumm1) {
+                            Date date = new GregorianCalendar(y2, ii-1, dayyy).getTime();
+
+                            if(sortedMap.containsKey(date)){
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' is not followed by whitespace.